### PR TITLE
Feature/gsye 564

### PR DESCRIPTION
**Pull request by hannesdiedrich**
## Reason for the proposed changes

Please describe what we want to achieve and why.

## Proposed changes

-

**INTEGRATION_TESTS_BRANCH**=feature/GSYE-564
**GSY_FRAMEWORK_BRANCH**=feature/GSYE-564

### CODE REVIEW

File: tests/strategies/test_strategy_infinite_bus.py [hunk 1]
```diff
@@ -316,7 +316,7 @@ def testing_event_market_cycle_post_offers(bus_test3, area_test3):
     bus_test3.event_market_cycle()
     assert len(area_test3.test_market.created_offers) == 1
     assert area_test3.test_market.created_offers[-1].energy == sys.maxsize
-    assert area_test3.test_market.created_offers[-1].price == float(30 * sys.maxsize)
+    assert area_test3.test_market.created_offers[-1].price - float(30 * sys.maxsize) == 0
```
Comment (spyrostz) [inline tests/strategies/test_strategy_infinite_bus.py:319]: Why not `isclose` ?

File: src/gsy_e/models/strategy/profile.py [hunk 2]
```diff
@@ -50,12 +50,7 @@ def read_or_rotate_profiles(self, reconfigure=False):
...
             profile = self.profile
 
-        profile = global_objects.profiles_handler.rotate_profile(
+        self.profile = global_objects.profiles_handler.rotate_profile(
             profile_type=self.profile_type,
             profile=profile,
             profile_uuid=self.input_profile_uuid)
-
-        if self.profile_type == InputProfileTypes.IDENTITY:
-            self.profile = convert_identity_profile_to_float(profile)
```
Comment (spyrostz) [inline src/gsy_e/models/strategy/profile.py:59]: Just for my understanding, the only reason for these lines was to support profiles with type `Dict[datetime, str]` ? If yes I am glad that we no longer accept such profiles

**Review by spyrostz — APPROVED** [review]
2 comments, LGTM though

File: tests/strategies/test_strategy_infinite_bus.py (hunk 1 above)
Comment (hannesdiedrich) [reply tests/strategies/test_strategy_infinite_bus.py:319]: I really do not have a clue. Will use is_close of course.

Comment (hannesdiedrich) [reply tests/strategies/test_strategy_infinite_bus.py:319]: I really do not have a clue. Will use is_close of course.

File: src/gsy_e/models/strategy/profile.py (hunk 2 above)
Comment (hannesdiedrich) [reply src/gsy_e/models/strategy/profile.py:59]: Exactly. When I found that out, I was also really relieved to remove it. :-)

**Pull request merged**